### PR TITLE
Removes some of the sphinx gallery ignore patterns.

### PR DESCRIPTION
**Pull request by arokem**

### CODE REVIEW

File: examples/tutorial_examples/baby_afq.py [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    Because it is time and disk-space consuming, this example
+    is not run when the pyAFQ documentation is built. To run this example
+    yourself, you can download the contents of this file as an
+    executable `.py` file or as a Jupyter notebook from the links at the bottom
+    of the page.
+
+"""
+import os.path as op
+import plotly
+import wget
```
Comment (chiuhoward) [inline examples/tutorial_examples/baby_afq.py:21]: Even development version of pyAFQ does not include wget module as part of package. Will need to specify somewhere or new user will be unable to import wget directly.

Comment (chiuhoward) [inline examples/tutorial_examples/baby_afq.py:21]: wget not packaged within development version of pyAFQ

**Comment by 36000** [comment]
I think wget is in pyAFQ now, on master at least. @arokem  is this ready to merge?

**Comment by arokem** [comment]
Let me do a quick rebase to make sure.

**Comment by arokem** [comment]
Yeah - I think this is ready, unless you have any suggestions.

**Comment by arokem** [comment]
Rebased on master, so assuming the CI passes with the astroid fix in place, this is ready for review and/or merge.

**Pull request merged**